playbook/app/pb_kits/playbook/index.js

```diff
@@ -93,7 +93,7 @@ export { default as StarRating } from './pb_star_rating/_star_rating'
 export { default as StatChange } from './pb_stat_change/_stat_change'
 export { default as StatValue } from './pb_stat_value/_stat_value'
 export { default as Table } from './pb_table/_table'
-export { default as TableRow } from './pb_table/_table_row'
+export { default as TableRow } from './pb_table/SubKits/_table_row'
```

Comment:
<img width="348" alt="Screenshot 2024-03-08 at 3 52 46 PM" src="https://github.com/powerhome/playbook/assets/9158723/7be36f83-6de9-4ef4-8788-cac215408b88">

Let's get a little more consistent with our naming style of these folders. 

Let's go with all lowercase. And use snakecase when necessary.

Reply:
Will do! Thank you!